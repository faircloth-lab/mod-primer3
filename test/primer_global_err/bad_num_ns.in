SEQUENCE_ID=bad_num_ns
SEQUENCE_TEMPLATE=TAGCTGATGCTAGCTAGCGTGGGCATGCTAGCTAGCTGATGCAGCTAGCTGATCGTAGCTAGCTATCGATCGTAGCTGACTGATCGTAGCTAGCTGATCGATGCTAGCTAGCTAGCTGATCATCTACGTAGCTGACTATCGATGCTACTACTATCGATCGTAGCTAGCTGATCGTAGCTAGCTAGCTGATGCTAC
PRIMER_NUM_NS_ACCEPTED=-1
=
