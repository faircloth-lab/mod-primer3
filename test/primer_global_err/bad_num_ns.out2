../src/primer3_core: Illegal value for PRIMER_NUM_NS_ACCEPTED
