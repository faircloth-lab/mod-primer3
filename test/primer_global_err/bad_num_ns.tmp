PRIMER_SEQUENCE_ID=bad_num_ns
SEQUENCE=TAGCTGATGCTAGCTAGCGTGGGCATGCTAGCTAGCTGATGCAGCTAGCTGATCGTAGCTAGCTATCGATCGTAGCTGACTGATCGTAGCTAGCTGATCGATGCTAGCTAGCTAGCTGATCATCTACGTAGCTGACTATCGATGCTACTACTATCGATCGTAGCTAGCTGATCGTAGCTAGCTAGCTGATGCTAC
PRIMER_NUM_NS_ACCEPTED=-1
PRIMER_ERROR=Illegal value for PRIMER_NUM_NS_ACCEPTED
=
