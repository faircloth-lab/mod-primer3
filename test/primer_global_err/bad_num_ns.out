SEQUENCE_ID=bad_num_ns
SEQUENCE_TEMPLATE=TAGCTGATGCTAGCTAGCGTGGGCATGCTAGCTAGCTGATGCAGCTAGCTGATCGTAGCTAGCTATCGATCGTAGCTGACTGATCGTAGCTAGCTGATCGATGCTAGCTAGCTAGCTGATCATCTACGTAGCTGACTATCGATGCTACTACTATCGATCGTAGCTAGCTGATCGTAGCTAGCTAGCTGATGCTAC
PRIMER_NUM_NS_ACCEPTED=-1
PRIMER_ERROR=Illegal value for PRIMER_NUM_NS_ACCEPTED
=
